cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -16,6 +15,14 @@
 
 from . import __version__
 
+
+chosen_config_path = None
```

Comment:
Why is it global?
All other configuration parameters are stored in `CherryPicker` class

Reply:
If I recall correctly it was the easiest way to get this change working without major refactoring which I've been trying to avoid. We can refactor it later once it's merged. All helper functions are separate from the class and they also need access to the config.
I was trying to stick to one logical change per PR principle.